generators/entity-server/templates/src/main/java/package/service/impl/_EntityServiceImpl.java

```diff
@@ -99,6 +97,17 @@
             .map(<%= entityToDtoReference %>);<% } %>
         <%_ } _%>
     }
+    <%_ if (fieldsContainOwnerManyToMany === true) { _%>
+
+    /**
+     * Get all the <%= entityClass %> with eager load of many-to-many relationships.
+     *
+     * @return the list of entities
+     */
+    public Page<<%= instanceType %>> findAllWithEagerRelationships(Pageable pageable) {
+        return <%= entityInstance %>Repository.findAllWithEagerRelationships(pageable);
+    }
+    <% } %>
```

Comment:
If the entity uses a DTO, then it needs to be mapped.  Replace the semicolon on line 108 with:
```
<% if (dto !== 'mapstruct') { %>;<% } else { %>
            .map(<%= entityToDtoReference %>);<% } %>
```